### PR TITLE
Mergesub

### PR DESCRIPTION
**Pull request by rdaly525**
@leonardt, I merged the add and subtract for floating point in this branch. I had to do a hack to get access to the concat operator in magma. Let me know if this is okay

Merging the add and subtract will get rid of the subtract bug in fpcomp2

### CODE REVIEW

**Comment by rdaly525** [comment]
@Kuree, could you take a look into this buildkite failure? For some reason it seems to be running a magma test and failing on an import error. Not sure why it is running a magma test...

File: .travis/run.sh [hunk 1]
```diff
@@ -3,4 +3,4 @@ export PYTEST_ADDOPTS="--color=yes"
 
 cd /lassen
 
-pytest -x -v tests/test_pe.py -k fp_binary
+pytest -x -v
```
Comment (Kuree) [inline .travis/run.sh]: You need to change it back to `tests/`

**Comment by Kuree** [comment]
@rdaly525 See my review comments.

File: .travis/run.sh (hunk 1 above)
Comment (rdaly525) [reply .travis/run.sh]: Thanks, just fixed

**Pull request merged**